base/applications/network/ping/ping.c

```diff
@@ -432,7 +432,13 @@ Ping(void)
             exit(1);
         }
 
-        ZeroMemory(SendBuffer, RequestSize);
+        /* Windows ping utility fills the optional data field with
+         * ASCII characters from 'a' to 'w', wrapping back around
+         * until SendBuffer is full. */
```

Comment:
Please, add a
`// TODO: Compare ReplyBuffer data to SendBuffer.`
before line
`486 492     free(SendBuffer);`